pkger/parser.go

```diff
@@ -575,6 +575,8 @@ func parseChart(r Resource) (chart, []failure) {
 		YPos:        r.intShort("yPos"),
 		Height:      r.intShort("height"),
 		Width:       r.intShort("width"),
+		Geom:        r.stringShort("geom"),
+		ShadeBelow:  r.boolShort("shadeBelow"),
```

Comment:
does this already exist above on line:
https://github.com/influxdata/influxdb/pull/15767/files#diff-bee826069937abe74ce78a93cc101d2fR571

if so then axe this